.github/mergify.yml

```
@@ -34,9 +38,11 @@ queue_rules:
 
   - name: low
     allow_inplace_checks: False
-    allow_checks_interruption: False
-    speculative_checks: 2
-    batch_size: 4
+    allow_checks_interruption: True
+    speculative_checks: 3
+    batch_size: 5
+    # Wait 1 hour to embark low priority tickets together in a merge train
+    batch_max_wait_time: 3600
```

Some PRs will merge in less than an hour, so I think it's enough to wait 10 minutes here.

```suggestion
    # Wait 10 minutes to embark low priority tickets together in a merge train
    batch_max_wait_time: 300
```